packages/client/components/Checkbox.tsx

```diff
@@ -1,29 +1,40 @@
-import React from 'react'
 import styled from '@emotion/styled'
+import {CheckBox, CheckBoxOutlineBlank, IndeterminateCheckBox} from '@mui/icons-material'
+import React from 'react'
 import {PALETTE} from '../styles/paletteV3'
-import Icon from './Icon'
-
 interface Props {
   active: boolean | null
   className?: string
   disabled?: boolean
   onClick?: (e: React.MouseEvent) => void
 }
 
-const StyledIcon = styled(Icon)<{disabled: boolean | undefined}>(({disabled}) => ({
+const StyledIcon = styled('div')<{disabled: boolean | undefined}>(({disabled}) => ({
   color: PALETTE.SLATE_600,
   cursor: disabled ? 'not-allowed' : 'pointer',
+  height: 24,
+  width: 24,
   display: 'block',
   opacity: disabled ? 0.38 : 1,
   userSelect: 'none'
 }))
 
 const Checkbox = (props: Props) => {
   const {active, className, disabled, onClick} = props
-  const icon = active ? 'check_box' : active === false ? 'check_box_outline_blank' : 'indeterminate_check_box'
+  const icon = active
+    ? 'check_box'
+    : active === false
+    ? 'check_box_outline_blank'
+    : 'indeterminate_check_box'
   return (
     <StyledIcon className={className} disabled={disabled} onClick={disabled ? undefined : onClick}>
-      {icon}
+      {
+        {
+          check_box: <CheckBox />,
+          check_box_outline_blank: <CheckBoxOutlineBlank />,
+          indeterminate_check_box: <IndeterminateCheckBox />
+        }[icon]
```

Comment:
-1 If all the icons are like this, you can put in a large `Icons` map object and take the values by icon type?

Reply:
Please leave the icon information local

Reply:
@Dschoordsch I don't get what you mean by local
Could you elaborate more about it?

Reply:
> all the icons are like this, you can put in a large Icons map object and take the values by icon type

@JimmyLv could you explain more how to do that?

Reply:
@87prashant this was a reply to @JimmyLv. I think what he meant (and I disagree with) is putting all icons in a separate file where we have a big map of the style
```
check_box: <CheckBox />,
...
```
Please don't do this.

Reply:
Ok, thanks